go.mod

```diff
@@ -55,6 +57,8 @@ require (
 	resenje.org/web v0.4.3
 )
 
+require github.com/ethersphere/batch-archive v0.0.2
```

Comment:
https://github.com/ethersphere/batch-archive/tree/main/archive should have only the gzip file. Both files are not needed and uncompressed one is quite large.